webapp/channels/src/components/custom_open_plugin_install_post_renderer/index.tsx

```diff
@@ -78,6 +80,7 @@ const ConfigureLink = (props: {pluginId: string; pluginName: string}) => {
             <Link
                 to={'/admin_console/plugins/plugin_' + props.pluginId}
                 style={{color: 'var(--denim-button-bg)', fontWeight: '600'}}
+                onClick={() => trackEvent('system_admin', 'system_console_visit', {location: 'custom_open_plugin_intall_post_renderer'})}
```

Comment:
Should this be `custom_open_plugin_install_post_renderer`? (changed intall to install)

Reply:
Thanks. Going to change it